src/app/route/NoMatch/NoMatch.container.js

```diff
@@ -31,15 +33,45 @@ export const mapDispatchToProps = (dispatch) => ({
             ({ default: dispatcher }) => dispatcher.update(breadcrumbs, dispatch)
         );
     },
+    updateMeta: (meta) => dispatch(updateMeta(meta)),
     changeHeaderState: (state) => dispatch(changeNavigationState(TOP_NAVIGATION_TYPE, state))
 });
 
 /** @namespace Route/NoMatch/Container/mapStateToProps */
 // eslint-disable-next-line no-unused-vars
-export const mapStateToProps = (state) => ({});
+export const mapStateToProps = (state) => ({
+    urlRewrite: state.UrlRewritesReducer.urlRewrite
+});
 
 /** @namespace Route/NoMatch/Container */
 export class NoMatchContainer extends PureComponent {
+    static propTypes = {
```

Comment:
Consider also removing this component from the app at all if it indeed is not necessary.

Reply:
I mean `NoMatchHandler` *